docs/guides/ai-transpiler-passes.mdx

```diff
@@ -102,6 +112,38 @@ The following custom collection passes for Cliffords, Linear Functions and Permu
 
 These custom collection passes limit the sizes of the collected sub-circuits so they are supported by the AI-powered synthesis passes. Therefore, it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
 
+## Hybrid Qiskit-AI circuit transpilation
+
+The `qiskit-ibm-transpiler` offers the option to configure a hybrid pass manager that combines the best of Qiskit and the AI-powered transpiler passes, so you can run a whole transpilation on your own. This feature behaves similarly to the Qiskit `generate_pass_manager` method. A typical way to use `generate_ai_pass_manager` is as follows:
```

Comment:
```suggestion
The `qiskit-ibm-transpiler` allows you to configure a hybrid pass manager that combines the best of Qiskit's heuristic and the AI-powered transpiler passes. This feature behaves similarly to the Qiskit `generate_pass_manager` method. A typical way to use `generate_ai_pass_manager` is as follows:
```

I took out `so you can run a whole transpilation on your own` because I don't understand what that means.

Reply:
Ok, I do agree it's more clear now. Thanks